File: src/main/webapp/js/service/cusorderService.js
//服务层
app.service('cusorderService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../cusorder/findAll');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../cusorder/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../cusorder/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../cusorder/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../cusorder/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../cusorder/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../cusorder/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: src/main/java/top/axbt/to/service/CusorderService.java
package top.axbt.to.service;
import java.util.List;

import top.axbt.to.domain.TbCusorder;
import top.axbt.to.entity.PageResult;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface CusorderService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbCusorder> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbCusorder cusorder);
	
	
	/**
	 * 修改
	 */
	public void update(TbCusorder cusorder);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbCusorder findOne(String id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(String[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbCusorder cusorder, int pageNum, int pageSize,int state);

	public PageResult findPage(TbCusorder cusorder, int pageNum, int pageSize);
}
